modules/microshift-install-greenboot-rpm.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="installing-greenboot-rpm_{context}"]
+== Installing the greenboot RPM
+
+//procedure needed
+
+The health check script is installed into the `/etc/greenboot/check/required.d` directory, but it is not automatically executed during the system boot. When the `greenboot-default-health-check` RPM is installed, it includes health check scripts verifying that DNS and OSTree services can be accessed.
```

Comment:
It's no longer the case. Once microshift-greenboot RPM is installed, it pulls the greenboot RPM, so the required scripts are always executed on boot.

Reply:
Please provide the procedure to install the microshift-greenboot RPM. I'm still thinking this is separate since it is optional.

Reply:
Yes, we already list MicroShift-specific health checks in the intridcution, using table format

Reply:
There's no procedure. One just needs to include it in the blueprint. 
Should we provide a blueprint snippet here?

Reply:
Let me know if you need my help in this.

Reply:
Yes, please. Do you have a snippet in the dev docs I can use?

Reply:
Provided the snippets offline to @ShaunaDiaz